src/platform/forms-system/src/js/web-component-patterns/fullNamePattern.js

```diff
@@ -160,7 +163,7 @@ const fullNameUI = (formatTitle, uiOptions = {}) => {
  * @returns {UISchemaOptions} uiSchema
  */
 const firstNameLastNameUI = (formatTitle, uiOptions = {}) => {
-  const uiSchema = {
+  return {
     ...fullNameNoSuffixUI(formatTitle, uiOptions),
```

Comment:
looks like this is meant to be `firstNameLastNameNoSuffixUI` here instead

Reply:
Fixed.  Thanks for catching this!